benchmark/benchmarks.jl

```diff
@@ -186,22 +194,28 @@ end
 SUITE["indices"] = BenchmarkGroup()
 
 function spmv32(A, x)
-    y = Tensor(Dense{Int32}(Element{0.0, Float64, Int32}()))
-    @finch (y .= 0; for i=_, j=_; y[i] += A[j, i] * x[j] end)
+    y = Tensor(Dense{Int32}(Element{0.0,Float64,Int32}()))
```

Comment:
In general, it would be good if we could keep code style refactoring in separate pull requests, as it makes the actual changes in this PR difficult to find.